### PR TITLE
CRW-6054 Update Idea to use rhel-els 9.2

### PR DESCRIPTION
**Pull request by SDawley**
Update dockerfiles to use rhel-els and update sync to not overwrite them with upstream versions.
Updating content sets for rhel 9

https://issues.redhat.com/browse/CRW-6054
https://issues.redhat.com/browse/CRW-3185

### CODE REVIEW

File: devspaces-idea/build/scripts/sync.sh [hunk 1]
```diff
@@ -118,24 +119,9 @@ cat << EOT > "${TARGETDIR}"/compatible-ide.json
...
-      io.openshift.expose-services="" \\
-      usage=""
-EOT
-echo "Converted Dockerfile"
+sed_in_place -r \
+  `# Update DevSpaces version for Dockerfile` \
+  -e "s/version=.*/version=\"$DS_VERSION\" \\\/" \
+  "${TARGETDIR}"/Dockerfile
+
+echo "Converted Dockerfile"
```
Comment (ibuziuk) [inline devspaces-idea/build/scripts/sync.sh:127]: no new line

File: devspaces-idea/content_sets.yml [hunk 2]
```diff
@@ -12,12 +12,11 @@
...
 s390x:
-- rhel-8-for-s390x-baseos-rpms
-- rhel-8-for-s390x-appstream-rpms
+- rhel-9-for-s390x-baseos-eus-rpms__9_DOT_2
+- rhel-9-for-s390x-appstream-eus-rpms__9_DOT_2
 ppc64le:
-- rhel-8-for-ppc64le-baseos-rpms
-- rhel-8-for-ppc64le-appstream-rpms
+- rhel-9-for-ppc64le-baseos-eus-rpms__9_DOT_2
+- rhel-9-for-ppc64le-appstream-eus-rpms__9_DOT_2
```
Comment (ibuziuk) [inline devspaces-idea/content_sets.yml:22]: no new line

File: devspaces-idea/build/scripts/sync.sh (hunk 1 above)
Comment (SDawley) [reply devspaces-idea/build/scripts/sync.sh]: Added new line

File: devspaces-idea/content_sets.yml (hunk 2 above)
Comment (SDawley) [reply devspaces-idea/content_sets.yml]: Added it

**Pull request merged**